jmh-benchmarks/src/main/java/org/apache/kafka/jmh/server/CheckpointBench.java

```diff
@@ -145,7 +145,7 @@ public void setup() {
         OffsetCheckpoints checkpoints = (logDir, topicPartition) -> Option.apply(0L);
         for (TopicPartition topicPartition : topicPartitions) {
             final Partition partition = this.replicaManager.createPartition(topicPartition);
-            partition.createLogIfNotExists(true, false, checkpoints);
+            partition.createLogIfNotExists(true, false, checkpoints, Option.empty());
```

Comment:
I think we can just not set the topic ID here, but want to confirm.

Reply:
Seems no harm either way, but why not set one anyway since that will be the default behavior going forward?